log/src/main/java/net/md_5/bungee/log/ConciseFormatter.java

```diff
@@ -27,6 +27,20 @@ public String format(LogRecord record)
         formatted.append( " [" );
         appendLevel( formatted, record.getLevel() );
         formatted.append( "] " );
+
+        if ( record.getLoggerName() != null && !record.getLoggerName().equals( "BungeeCord" ) )
```

Comment:
bungee-log is meant to be used in other projects, so hardcoding BungeeCord is not ideal

Reply:
Yes but i found no better way to do this.
Do you have a better Ideal?
I think we would use the logrecord paramdters or something